terraform/hub/README.md

```diff
@@ -0,0 +1,161 @@
+# ArgoCD on Amazon EKS
+
+This pattern shows how to use the new stack platform concept
+
+This tutorial guides you through deploying an Amazon EKS cluster with addons configured via ArgoCD, employing the [GitOps Bridge Pattern](https://github.com/gitops-bridge-dev).
+
+
+## Prerequisites
+Before you begin, make sure you have the following command line tools installed:
+- git
+- terraform
+- kubectl
+- argocd
+
+## Fork the Git Repositories
+
+### Fork the Addon GitOps Repo
+1. Fork the git repository for addons [here](https://github.com/gitops-bridge-dev/gitops-bridge-argocd-control-plane-template).
```

Comment:
Why adding this instructions that are for the GitOps bridge repo ?

Reply:
Good point i kept the old readme i will remove that